### PR TITLE
ECL-697: Cancel amendment

### PR DESCRIPTION
**Pull request by paddynski-opencast**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -24,7 +24,7 @@ play.filters.csp {
     directives {
         base-uri = "'self'"
         connect-src = "'self' https://www.google-analytics.com https://region1.google-analytics.com"
-        form-action = "'self'"
+        form-action = "'self' http://localhost:14008"
```
[the remark this answers is not in the document]
Comment (TheDanielDoyle) [reply conf/application.conf:27]: You are only allowing form action to a service you specify. In this case, ECL Account Frontend

File: app/uk/gov/hmrc/economiccrimelevyreturns/views/AmendReturnSubmittedView.scala.html [hunk 2]
```diff
@@ -17,14 +17,16 @@
...
+@import uk.gov.hmrc.economiccrimelevyreturns.viewmodels.AddressViewModel
 
 @this(
         layout: templates.Layout,
         govukPanel: GovukPanel,
         appConfig: AppConfig
 )
 
-@(fyStart: LocalDate, fyEnd: LocalDate, confirmationEmail: String)(implicit request: Request[_], messages: Messages)
+@(fyStart: LocalDate, fyEnd: LocalDate, confirmationEmail: String, contactAddress: Option[EckAddress])(implicit request: Request[_], messages: Messages)
```
[the remark this answers is not in the document]
Comment (paddynski-opencast) [reply app/uk/gov/hmrc/economiccrimelevyreturns/views/AmendReturnSubmittedView.scala.html]: Fixed. Some of the preliminary work I did for ECL-737 had to be removed!

File: conf/application.conf (hunk 1 above)
[the remark this answers is not in the document]
Comment (liandra-giacomine) [reply conf/application.conf:27]: But what if someone is running a service on locally on that port?

[the remark this answers is not in the document]
Comment (TheDanielDoyle) [reply conf/application.conf:27]: It's overriden in each environment config.

**Review by TheDanielDoyle — APPROVED** [review]

**Pull request merged**